app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt

```diff
@@ -1,44 +1,63 @@
 package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
 
+import androidx.compose.foundation.gestures.detectTapGestures
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.platform.LocalSoftwareKeyboardController
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.kks.nimblesurveyjetpackcompose.R
-import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.CHOICE
-import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.DROPDOWN
-import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.NONE
-import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.NPS
-import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.SMILEY
-import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.STARS
-import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.THUMBS
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.*
 import com.kks.nimblesurveyjetpackcompose.model.SurveyAnswer
 import com.kks.nimblesurveyjetpackcompose.model.SurveyQuestion
 import com.kks.nimblesurveyjetpackcompose.model.SurveyQuestionPickType
-import com.kks.nimblesurveyjetpackcompose.ui.presentation.common.SurveyBoldText
+import com.kks.nimblesurveyjetpackcompose.ui.presentation.common.SurveyText
 import com.kks.nimblesurveyjetpackcompose.ui.theme.White50
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
 
 private const val NUMBER_OF_EMOJI_ANSWERS = 5
+private const val DELAY_TO_CLEAR_FOCUS = 500L
 
+@OptIn(ExperimentalComposeUiApi::class)
 @Composable
 fun SurveyQuestionScreen(
     surveyQuestion: SurveyQuestion,
     pageNumber: Int,
     totalNumberOfPage: Int,
     onChooseAnswer: (questionId: String, surveyAnswers: List<SurveyAnswer>) -> Unit
 ) {
+    val keyboardController = LocalSoftwareKeyboardController.current
+    val focusManager = LocalFocusManager.current
+    val coroutineScope = rememberCoroutineScope()
```

Comment:
Shall we apply this to the `Box` in `SurveyDetailScreen` instead? Tapping on the area above the question (for example, to the right of the `(x)` button) doesn't remove the focus 👀